### DISCUSSION TITLE
QRControl UTC Time

### DISCUSSION THREAD

**Discussion by catch-point** [gopro/labs#124] [Feature Requests]
It would be nice if QRControl made it easy to switch between local and UTC time zones.

Most of the time I set my GoPro Hero10 Black to local time as this is the industry standard. Quik and other applications assume the timestamp is local time. However, GoPro Cloud assumes UTC time. So, if I plan on uploading them directly to GoPro Cloud first, I'll set the GoPro time to UTC.

For example, if I'm using the motion detection trigger with auto upload, I'll use the webpage [precisiontime_utc](https://gopro.github.io/labs/control/precisiontime_utc/) to switch the GoPro to UTC then using QRControl to start motion capture w/upload like this `!SM3D4H5!U!R`.

By setting the GoPro to UTC before capture when using `!U`, GoPro Cloud has the correct time and any downloads from GoPro Cloud also have the correct time (including Quik). Whereas if I'm transferring via USB, I'll want GoPro to capture using local time to get the correct time in the files.

**Comment by fxstein** [comment]
FYI The GPS time in your media is always UTC.

I am working on a simple feature in [GoProX](https://github.com/fxstein/GoProX) that lets you select GPS, UTC or local time after the fact for all GoPro media. Not done with it yet, but have the geocode lookups built to allow to converts any UTC time into local time for the GoPro media.

I then upload my GoPro media after the processing to the GoPro Cloud and can use it on the go in Quik.
